### PR TITLE
add banner to readme

### PR DESCRIPTION
**Pull request by lucakiebel**
### What did you fix? (provide a description or issue closes statement)

added banner to readme.md

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[No]**
- Have I run the linter? **[No]**
- Is is a bug fix, feature request, or enhancement? **[Enhancement]**

### CODE REVIEW

**Comment by TobiTenno** [comment]
you'll need to submit a PR to https://github.com/WFCD/banner first

**Comment by TobiTenno** [comment]
also, typically just do either badge or banner, not both :D

**Comment by lucakiebel** [comment]
The usage guide on the banner repo is a little misleading there:

> To use the banner, please submit a pull request to this repository adding an entry to PROJECTS.md, providing proof that you are using the below snippet in your readme appropriately.

I can't proof that I am using the banner in my readme without putting it in there first 😅😅😅

**Comment by TobiTenno** [comment]
Ok, sure, either way... You only need one or the other, but both banner and badge

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # Warframe.js
+
+[![Supported by Warframe Community Developers](https://warframestat.us/wfcd.png)](https://github.com/WFCD "Supported by Warframe Community Developers")
```
Comment (TobiTenno) [inline README.md]:
```suggestion
```

**Review by TobiTenno — CHANGES_REQUESTED** [review]

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**